content_schemas/formats/shared/definitions/_whitehall.jsonnet

```
@@ -417,15 +417,27 @@
           summary: {
             "$ref": "#/definitions/promotional_feature_item_summary",
           },
-          youtube_video_id: {
-            type: "string",
-          },
           double_width: {
             "$ref": "#/definitions/promotional_feature_item_double_width",
           },
           links: {
             "$ref": "#/definitions/promotional_feature_item_links",
           },
+          youtube_video: {
+            type: "array",
```

I think this is supposed to be an [object][] rather than an [array][], since we've defined properties on it (which is something JSON objects have, not JSON arrays).

[object]: https://json-schema.org/understanding-json-schema/reference/object.html
[array]: https://json-schema.org/understanding-json-schema/reference/array.html

```suggestion
            type: "object",
```

---

Good catch. Have updated. Thanks!